website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md

```diff
@@ -78,6 +78,34 @@ As example:
 dbt build --select result:error+ result:fail+
 ```
 
+## Fresh Rebuilds
```

Comment:
I felt this wanted a header, for the version-block logic to make sense. This is a total placeholder. What speaks to you?